modules/tfm/tfm/boards/CMakeLists.txt

```diff
@@ -35,3 +35,31 @@ if(BL2)
     PRIVATE ${board_includes}
     )
 endif()
+
+if(TFM_PARTITION_PLATFORM)
+  set(src ${CMAKE_CURRENT_LIST_DIR}/src)
```

Comment:
minor nit, should it be `src_dir` now that the other variable is named `nrf_dir` ?